src/targets.rs

```diff
@@ -897,6 +901,7 @@ mod tests {
             "wasm32-experimental-emscripten",
             "wasm32-unknown-emscripten",
             "wasm32-unknown-unknown",
+            "wasm32-unknown-wasi",
```

Comment:
As of https://github.com/rust-lang/rust/pull/60585 (landed yesterday), this triple is now named "wasm32-wasi".

This will also need `Triple`'s `fmt` to omit the vendor in this case (as it already does in several other cases) so that it can round-trip "wasm32-wasi" from string to `Triple` back to string.